esmvaltool/diag_scripts/validation.py

```diff
@@ -187,9 +245,17 @@ def plot_ctrl_exper(ctrl, exper, cfg, plot_key):
 
 
 def plot_ctrl_exper_seasons(ctrl_seasons, exper_seasons, cfg, plot_key):
-    """Call plotting functions and make plots with seasons"""
+    """Call plotting functions and make plots with seasons."""
     seasons = ['DJF', 'MAM', 'JJA', 'SON']
-    if cfg['analysis_type'] == 'zonal_mean':
+    if cfg['analysis_type'] == 'lat_lon':
+        for c_i, e_i, s_n in zip(ctrl_seasons, exper_seasons, seasons):
```

Comment:
`c_i` is still not human readable, even if it doesn't give you a linter error. Maybe you can come up with some variable names that are easy to understand for humans?

Reply:
yeah good point, will have them names nicely in mate